README.md

```diff
@@ -87,6 +100,29 @@ cargo build --workspace
 cargo test --workspace
 ```
 
+### Using Podman instead of Docker
+
+If you want to use podman you have to enable podman in Visual Studio Code like
+described [here](https://code.visualstudio.com/remote/advancedcontainers/docker-options#_podman)
+and update the `.devcontainer/devcontainer.json` file with the following additions:
+
+```Dockerfile
```

Comment:
But this is snippets of a Dockerfile and therefore probably not something that a syntax highlighter will be happy with so may be remove this?

Reply:
Originally, I wanted to place an alternative Dockerfile into the `.devcontainer/` folder. but decided to give at least the corresponding hints on what to change if someone wants to use it with Podman. The snippet should just help getting the right changes, maybe splitting them into separate chunks?

Reply:
Included your suggestion using jsonc as code fence block syntax